event_sale_reservation/wizards/registration_editor.py

```diff
@@ -47,7 +48,11 @@ def action_convert_to_registration(self):
             "actions": [
                 {"type": "ir.actions.act_window_close"},
                 {
-                    "context": dict(self.env.context, registering_reservations=False),
+                    "context": dict(
+                        self.env.context,
+                        registering_reservations=False,
+                        skip_event_sale_registration_multi_qty=True,
```

Comment:
If we go this way, give at least an explanation of why this context key is needed. Anyway. This doesn't make both modules compatible, so maybe a note would be needed in the ROADMAP as well